### ISSUE TITLE
Flatpak error when trying to run Musescore

### ISSUE THREAD

**Issue by animeshadowpanda** [flatpak/flatpak#4631]
Hey there, this the error I get:
`flatpak run org.musescore.MuseScore
bwrap: Can't mount proc on /newroot/proc: Operation not permitted
error: ldconfig failed, exit status 256`

I am on ChromeOS 96.0.4664.111, and Flatpak version 1.10.5.

**Comment by Ratevandr** [comment]
This bug report should be opened here: :https://github.com/flathub/org.musescore.MuseScore

**Comment by smcv** [comment]
> This bug report should be opened here: :https://github.com/flathub/org.musescore.MuseScore

Not necessarily: this looks more likely to be a general problem with running Flatpak apps on this particular machine (or on ChromeOS in general).

@animeshadowpanda, please try running a different Flatpak app like `org.openttd.OpenTTD` or `org.kde.minuet` to confirm whether this is a problem that is specific to Musescore, or a general problem with all Flatpak apps?

> I am on ChromeOS 96.0.4664.111

I believe we've had issues in the past as a result of ChromeOS not being a typical Linux distribution.

We will need more information to have any chance of solving this. Please see https://github.com/flatpak/flatpak/blob/master/.github/ISSUE_TEMPLATE/bug_template.yml for the sort of information we need.

Other things that would be useful to know:

Are you running Flatpak in some sort of container, chroot, or other environment that is not the "real" operating system?

What is the output if you run `flatpak run -vv org.musescore.MuseScore`?

**Comment by smcv** [comment]
> Are you running Flatpak in some sort of container, chroot, or other environment that is not the "real" operating system?

The answer appears to be yes: the typical way to run Flatpak on ChromeOS is to run it under Crostini, which uses lxd containers.

If this is the same problem as in #3644, then this might help: https://www.reddit.com/r/Crostini/comments/rvjsi3/flatpak_apps_dont_work_in_chrome_os/hrbh6go/?utm_source=reddit&utm_medium=web2x&context=3

ChromeOS has some weird requirements that are not present on normal Unix systems. Try this: https://www.reddit.com/r/Crostini/comments/rvjsi3/flatpak_apps_dont_work_in_chrome_os/hrbh6go/?utm_source=reddit&utm_medium=web2x&context=3

**Comment by Newbytee** [comment]
This also happens reproduces when trying to build Flatpak packages with flatpak-builder in Bitbucket CI, maybe related to this: https://jira.atlassian.com/browse/BCLOUD-13420.

Unfortunately the suggestion from Reddit isn't applicable in this case.

**Comment by smcv** [comment]
> This also happens reproduces when trying to build Flatpak packages with flatpak-builder in Bitbucket CI, maybe related to this: https://jira.atlassian.com/browse/BCLOUD-13420.

If `CAP_SYS_ADMIN` is not available, then bubblewrap and therefore Flatpak cannot work. There is nothing that bubblewrap or flatpak can do to solve this: some container environments just cannot run them.

> Is this worth opening a separate issue for?

Strictly speaking it's a separate issue, but we'd have to close it immediately as "won't fix" (or more accurately "can't fix"), so there isn't much point in opening it.
